.github/ISSUE_TEMPLATE/1_add-axis.md

```diff
@@ -0,0 +1,51 @@
+---
+name: Add Axis
+about: Submit a custom Axis Google Fonts
```

Comment:
Can probably remove 'Google Fonts' and drop Axis to lowercase here.  If you prefer to keep the GF in the about, I suggest moving it before the word axis.

Reply:
I missed the "to" preposition. It would be: Submit a custom axis to Google Fonts Axis Registry.
Sounds good?

Reply:
Sounds good to me!